portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,24 @@ function openTab(evt, tabName) {
     evt.currentTarget.classList.add("active");
 }
 
+
+
+function hideButton(id) {
+    var x = document.getElementById(id);
+    if (x.style.display === "none") {
+    x.style.display = "block";
+    } else {
+    x.style.display = "none";
+  }
+}
+
+function fixSite() {
+    var html = document.getElementById("html");
```

Comment:
You can use getElementByTagName('html')[0] instead, since there will only ever be 1 html element on the page.

Even better, you can use `document.body`, and add the styling to that instead.  I think its usually better to style the body rather than the html element

Reply:
Just fixed this with the latest round of commits! Thanks for the feedback.